pkg/setup-isoreboot/dell.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os/exec"
+	"time"
+
+	"github.com/cybozu-go/well"
+)
+
+const idracadm7Path = "/opt/dell/srvadmin/bin/idracadm7"
+
+func setupDell(ctx context.Context, url string) error {
+	err := well.CommandContext(ctx, idracadm7Path, "vmdisconnect").Run()
+	// we cannot use exit status to detect errors because `idracadm7 vmdisconnect` returns nonzero status if it is not connected.
+	var exitError *exec.ExitError
+	if err != nil && !errors.As(err, &exitError) {
+		return fmt.Errorf("racadm vmdisconnect failed: %w", err)
+	}
+
+	err = well.CommandContext(ctx, idracadm7Path, "remoteimage", "-d").Run()
```

Comment:
I suppose you have confirmed that "idracadm remoteimage -d" does not return an error even when a remote image is not connected.
This behavior seems non-intuitive, especially after the comment for "idracadm vmdisconnect."  So I suggest commenting here.